### PR TITLE
AK-17458 Add update support for connector_internet_exit

### PR DESCRIPTION
**Pull request by spikewang**
Add update support for connector_internet_exit and rename the connector to
"internet_exit" to be consistent with portal.

### CODE REVIEW

**Comment by spikewang** [comment]
I have created config for some basic testings including updating segment.

**Review by bchakrav1 — APPROVED** [review]
Looks good

File: alkira/provider.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func Provider() *schema.Provider {
 			"alkira_connector_aws_vpc":       resourceAlkiraConnectorAwsVpc(),
 			"alkira_connector_azure_vnet":    resourceAlkiraConnectorAzureVnet(),
 			"alkira_connector_gcp_vpc":       resourceAlkiraConnectorGcpVpc(),
-			"alkira_connector_internet":      resourceAlkiraConnectorInternet(),
+			"alkira_connector_internetExit":  resourceAlkiraConnectorInternetExit(),
```
Comment (rvichery) [inline alkira/provider.go:40]:
```suggestion
			"alkira_connector_internet_exit":  resourceAlkiraConnectorInternetExit(),
```

**Review by rvichery — CHANGES_REQUESTED** [review]
Probably just a typo on the resource name. Also we should update the internet connector examples to match this change.

File: alkira/resource_alkira_connector_internet_exit.go [hunk 2]
```diff
@@ -85,20 +78,49 @@ func resourceConnectorInternetCreate(d *schema.ResourceData, m interface{}) erro
...
+func resourceConnectorInternetExitUpdate(d *schema.ResourceData, m interface{}) error {
+	client := m.(*alkira.AlkiraClient)
+
+	connector, err := generateConnectorInternetRequest(d, m)
+
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[INFO] Updateing Connector (INTERNET) %s", d.Id())
```
Comment (rvichery) [inline alkira/resource_alkira_connector_internet_exit.go:97]:
```suggestion
	log.Printf("[INFO] Updating Connector (INTERNET-EXIT) %s", d.Id())
```

File: alkira/resource_alkira_connector_internet_exit.go [hunk 3]
```diff
@@ -85,20 +78,49 @@ func resourceConnectorInternetCreate(d *schema.ResourceData, m interface{}) erro
...
+	err = client.UpdateConnectorInternetExit(d.Id(), connector)
+
+	return err
 }
 
-func resourceConnectorInternetDelete(d *schema.ResourceData, m interface{}) error {
+func resourceConnectorInternetExitDelete(d *schema.ResourceData, m interface{}) error {
 	client := m.(*alkira.AlkiraClient)
 
 	log.Printf("[INFO] Deleting Connector (internet-exit) %s", d.Id())
```
Comment (rvichery) [inline alkira/resource_alkira_connector_internet_exit.go:106]: To match other resources
```suggestion
	log.Printf("[INFO] Deleting Connector (INTERNET-EXIT) %s", d.Id())
```

**Review by rvichery — APPROVED** [review]
Looks good, thanks for making the changes!

**Pull request merged**